parsers/markdown.c

```diff
@@ -57,6 +60,8 @@ static kindDefinition MarkdownKinds[] = {
 	{ true, 'T', "l4subsection",  "level 4 sections" },
 	{ true, 'u', "l5subsection",  "level 5 sections" },
 	{ true, 'n', "footnote",      "footnotes" },
+	{ true, 'h', "hashtag",       "hashtags"},
+	{ true, 'H', "subtag",       "subtags in hashtags"},
```

Comment:
Do you think we should have both hashtag and subtag?
Having only hashtag is not enough?